environment/README.md

```diff
@@ -0,0 +1,21 @@
+## Set up Cromwell configuration for running on Google Cloud
+
+```
+$ python create_google_config.py
+```
+
+<!-- To run this script:
+```
+./run.sh environment/create_google_config.py
+``` -->
+
+<!--To run using Docker:
+```
+docker run \-\-rm -it -v "$HOME"/.config:/.config broadinstitute/firecloud-tools python /environment/create_google_config.py
+```-->
+
+## Prerequisites
```

Comment:
are there any python libraries that someone would need to install to run this when not using docker?  if so, we should setup a way to install those